scripts/export_sqlite_to_workspace.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python
```

Comment:
This should have a copyright statement (I think this would be GPL although I prefer Apache2). It should also have a comment to describe what this does.

Usually it would be one of the three-quote comments (""")